### DISCUSSION TITLE
Design Document for NammaYatri

### DISCUSSION THREAD

**Discussion by divyankachaudhari** [nammayatri/nammayatri#4011] [Product Roadmap]
I'm a student from IIT Delhi, working on the spatio-temporal fairness of ride-hailing apps and potential improvements. I'm researching the current methods used so they could be benchmarked against for a more open and fair driver/customer experience. Are there any resources I could refer to? The code is difficult to understand without a design or a supporting document. The READMEs aren't enough, and some are also outdated. 

I'm particularly interested in the 'Dynamic Allocator' Part.

**Comment by kamikazeeGodzilla** [comment]
They just open sourced it that's it.
